### PR TITLE
Addtion of JIT provisioning

### PR DESCRIPTION
**Pull request by checklyalex**
## Affected Components
* [ ] Content & Marketing
* [ ] Test
* [x] Docs
* [ ] Learn
* [ ] Other

### CODE REVIEW

File: site/content/docs/single-sign-on/saml.md [hunk 1]
```diff
@@ -22,13 +20,13 @@ Setting up SSO for your organisation starts with providing [Checkly Support](mai
 
 ## IdP configuration
 
-After configuration has taken place on Checkly's side, you will receive the following information to configure your IdP:
+After your configuration has been processed Checkly's side, you'll receive the following information to configure your IdP:
```
Comment (ndom91) [inline site/content/docs/single-sign-on/saml.md:23]:
```suggestion
After your configuration has been processed on Checkly's side, you'll receive the following information to configure your IdP:
```

File: site/content/docs/single-sign-on/saml.md [hunk 2]
```diff
@@ -22,13 +20,13 @@ Setting up SSO for your organisation starts with providing [Checkly Support](mai
...
 
 - a redirect URL (e.g. `https://auth.checklyhq.com/login/callback?connection=<YOUR_CONNECTION_ID>`)
 - a signout URL (normally `https://auth.checklyhq.com/logout`)
 - XML metadata (e.g. `https://auth.checklyhq.com/samlp/metadata?connection=<YOUR_CONNECTION_ID>`)
 
-Please make sure that Checkly is sent the email address back from your SSO Idp. This way Checkly can map the existing Checkly user to your Idp user. The object returned should have an “email” field, e.g.:
+Ensure Checkly is sent the correct email address back from your SSO Idp. Checkly can then map the existing user to your Idp user. The object returned should have an “email” field, e.g.:
```
Comment (ndom91) [inline site/content/docs/single-sign-on/saml.md:29]:
```suggestion
Ensure your SSO IdP is sending Checkly the correct email address. Checkly can then map the existing user to your IdP user. The object returned should have an “email” field, e.g.:
```

File: site/content/docs/single-sign-on/saml.md [hunk 3]
```diff
@@ -41,10 +39,35 @@ Please make sure that Checkly is sent the email address back from your SSO Idp.
...
-Once that is taken care of, you should be able to log in to Checkly via SSO already. Entering an email address associated with the domain you have provided in the login prompt should result in the password field disappearing:
+Once that is taken care of, logging in to Checkly via SSO is ready. Entering an email address associated with the domain you have provided in the login prompt should result in the password field disappearing:
 
 ![checkly login prompt without password screenshot](/docs/images/single-sign-on/checkly-login-prompt-sso.png)
 
 After submitting the Checkly login form, you should be redirected to your SSO login interface. Completing the login procedure will then lead you to your existing Checkly account, if you have one, or to the new account creation screen, in case you don't.
 
-> Note: once SAML SSO has been set up, you will still need to invite new users from your organization to your Checkly account, as they will not be added automatically.
\ No newline at end of file
+> Note: once SAML SSO has been set up, you will still need to invite new users from your organisation to your Checkly account, as they will not be added automatically.
```
Comment (ndom91) [inline site/content/docs/single-sign-on/saml.md:48]: They don't need to be "invited", per se. They just need to be assigned access to the "Checkly App" in their SSO dashboard. 

i.e. in Okta you have a list of all the apps your users can possibly use / that have been setup with your company. Then the admin has to "assign access" to the apps to each user (or group or whatever).

At that point, once they have access via their IdP, they can just login to Checkly and be added automatically to their respective Checkly Account ("organisation").

**Review by ndom91 — CHANGES_REQUESTED** [review]
A few small suggestions, but otherwise LGTM

File: site/content/docs/single-sign-on/saml.md (hunk 3 above)
Comment (checklyalex) [reply site/content/docs/single-sign-on/saml.md:48]: Do you think I should remove the note completely? 
So if a customer sets up SSO, they don't have to invite users from Checkly? What happens if they do?

Comment (ndom91) [reply site/content/docs/single-sign-on/saml.md:48]: Correct, they do not need to invite them. If they do, the user will still be added to their companies account with whichever method they used to signin (most likely their SSO IdP in this scenario, right). 

I think you can probably remove this note entirely actually. The general flow of setup your SSO connection with checkly and your users can signin and be auto provisioned is explained elsewhere thoroughly enough on this page, right. 

What do you think?

Comment (checklyalex) [reply site/content/docs/single-sign-on/saml.md:48]: Agreed :D

**Review by ndom91 — APPROVED** [review]
LGTM 👍

**Pull request merged**